prometheus/README.md

```diff
@@ -0,0 +1,33 @@
+# Prometheus Monitoring - BOSH deployment details
+
+Prometheus is an Open source monitoring tool.  A BOSH release was created for it [prometheus-boshrelease](https://github.com/cloudfoundry-community/prometheus-boshrelease).
+
+The BOSH release deploys the following monitoring tools - [Prometheus](<https://prometheus.io/>), [AlertManager](https://github.com/prometheus/alertmanager) and  [Grafana](<https://grafana.com/>).
+
+
+## Usage
+
+### Setting up local BOSH environment
+The instructions for deploying a local BOSH2 director are [here](../setup-local-bosh.md)
+
+### Basic deployment
+The [prometheus boshrelease](https://github.com/cloudfoundry-community/prometheus-boshrelease) provides great documentation as well as the core BOSH deployment manifest and standard operator files.  
+This *DTA* repository provides some custom operator files and a concourse pipelines for deploying it to our environments.
+
+#### [aws-environment](operators/aws-environment.yml)
+This operator file performs the main customisations for deploying prometheus to cloud.gov.au environments.
+
+#### [deployment-name](operators/deployment-name.yml)
+The default deployment name `prometheus` is specified in the main deployment manifest.  This operator file allows you change the name - useful if you deploy more than 1 prometheus service.
+
+#### [networking](operators/networking.yml)
+Update the network name from `default` to `Support`
```

Comment:
Is setting the network name `Support` too specific? Maybe let the operator pass in the name via `-v`?

Reply:
I'm fine with being specific until we need something general, but I think that if we're going to be this specific, we might as well have a "dta-platform" (or whatever) ops file for all things specific to our platform, instead of one ops file for the network and another for the VM extension and another for the VM sizes, etc.  (It's possible to override parts of an ops file with a later ops file, so we don't lose any hackability in dev.)

Reply:
Yep I think we're thinking the same thing - aws-environment.yml could be renamed to dta-platform.yml https://github.com/AusDTO/bosh-deployments/pull/15#discussion_r141794532

Reply:
i initially thought having many ops files was good, but then it became hard to manage and i dumped most of the customisations into 1 file.  i'll rename `aws-environment.yml` to reflect `dta-platform.yml` as that's more descriptive of what it is.